src/metabase/pulse.clj

```diff
@@ -93,6 +112,26 @@
   [results]
   (every? is-card-empty? results))
 
+(defn- cards->notifications [{:keys [cards channel-ids], :as pulse}]
```

Comment:
does it make more sense to rename this function to `pulse->notifications`? It looks like its param is a Pulse, not a sequence of Cards. For clarity

Reply:
I agree, I'll change it